upstream_utils/update_stack_walker.py

```diff
@@ -14,19 +14,18 @@
 def main():
     root, repo = setup_upstream_repo(
         "https://github.com/JochenKalmbach/StackWalker",
-        "42e7a6e056a9e7aca911a7e9e54e2e4f90bc2652")
+        "42e7a6e056a9e7aca911a7e9e54e2e4f90bc2652",
+        shallow=False)
     wpiutil = os.path.join(root, "wpiutil")
 
-    pr35_url = "http://patch-diff.githubusercontent.com/raw/JochenKalmbach/StackWalker/pull/35.patch"
-    pr35_patch_download_path = os.path.join(tempfile.gettempdir(),
-                                            "stackwalker.patch")
-
-    response = urllib.request.urlopen(pr35_url)
```

Comment:
Since these aren't used anymore, a lot of standard library imports can be removed.

Reply:
If you like it then you should have put a [linter](https://pypi.org/project/autoflake/) on it